src/use-swr-infinite.ts

```diff
@@ -67,7 +67,8 @@ function useSWRInfinite<Data = any, Error = any>(
   )
   let {
     initialSize = 1,
-    revalidateAll = false,
+    // should revalidate pages when a refresh timer is invoked
+    revalidateAll = !!config.refreshInterval,
```

Comment:
I'm not sure if we want to do this. Normally the `useSWRInfinite` API scenarios are cursor based, and in those cases we only need to revalidate the first page when polling because updates will likely happen in the newest page, and it will be too expensive to always revalidate all. Maybe it's better to let the user to enable it explicitly.